web/components.d.ts

```diff
@@ -45,7 +45,7 @@ declare module '@vue/runtime-core' {
     IMdiClockTimeEightOutline: typeof import('~icons/mdi/clock-time-eight-outline')['default']
     IMdiFormatListBulleted: typeof import('~icons/mdi/format-list-bulleted')['default']
     IMdiGestureTap: typeof import('~icons/mdi/gesture-tap')['default']
-    IMdiGithub: typeof import('~icons/mdi/github')['default']
+    IMdiGitHub: typeof import('~icons/mdi/github')['default']
```

Comment:
This might be an API break...

Reply:
this file is generated so can not be changed manually

Reply:
Dropped